src/blenderbim/blenderbim/bim/module/pset/qto_calculator.py

```diff
@@ -175,3 +178,63 @@ def get_volume(self, o, vg_index=None):
 
                 volume += v1.dot(v2.cross(v3)) / 6.0
         return volume
+
+def angle_between_vectors(v1, v2):
+    return np.arccos(np.clip(np.dot(v1, v2), -1.0, 1.0))
+
+def calculate_net_lateral_area(object, angle_a=45, angle_b=135):
```

Comment:
`object` overrides the builtin type, you can use `obj` instead, it's widely recognized in blender addons